packages/web/src/components/cover-photo/ServerCoverPhoto.tsx

```diff
@@ -0,0 +1,31 @@
+import { memo } from 'react'
+
+import { Nullable } from '@audius/common/utils'
+import cn from 'classnames'
+import { FileWithPreview } from 'react-dropzone'
+
+import styles from './CoverPhoto.module.css'
+
+type CoverPhotoProps = {
+  userId: Nullable<number>
+  updatedCoverPhoto?: string
+  className?: string
+  loading?: boolean
+  error?: boolean
+  edit?: boolean
+  darken?: boolean
+  onDrop?: (
+    file: FileWithPreview[],
+    source: 'original' | 'unsplash' | 'url'
+  ) => Promise<void>
+}
+
+const ServerCoverPhoto = ({ className }: CoverPhotoProps) => {
+  return (
+    <div className={cn(styles.coverPhoto, className)}>
+      {/* TODO: Add static image here */}
```

Comment:
We don't render cover photos on the server side, so this is placeholder until whenever that work is done